lib/floe/workflow/context.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Floe
+  class Workflow
+    class Context
+      def initialize(context = nil, input: {})
+        context = JSON.parse(context) if context&.kind_of?(String)
+
+        @context = context || {
+          "Execution"    => {
+            "Input" => input
+          },
+          "State"        => {},
+          "States"       => [],
```

Comment:
NOTE `States` doesn't exist in the AWS Step Functions version but we need it to show the execution history.  I'm just appending the current `"State"` to the list of `"States"` after the step execution is finished.